web-dev/src/components/BoxArtList/BoxArt.js

```diff
@@ -0,0 +1,17 @@
+import BoxArtItem from "./BoxArtItem.js";
+import Movie from "../data/Movie.json";
```

Comment:
Movie data is only filler data and will be replaced by back end DAO/Services etc etc